modules/update-service-overview.adoc

```diff
@@ -1,7 +1,7 @@
 // Module included in the following assemblies:
 //
 // * architecture/architecture.adoc
-// * upgrading/upgrading-cluster.adoc
+// * updating/updating-cluster.adoc
```

Comment:
@abhinavdahiya, does this conceptual information about updates still sound right to you, or does it need to change?